### ISSUE TITLE
The accuracy suddenly became very low

### ISSUE THREAD

**Issue by 408550969** [weiliu89/caffe#762]
Recently, the training accuracy becomes very low when training(55%before ,and22% after), I did't change  the network and the data .
I try to reinstall the system, but it is still.Yesterday,When I cover the data sets from other computers, it becomes normal. But after training once, it's abnormal again. Half a month ago, everything was ok. Has anyone ever been in this situation?

**Comment by RoseLii** [comment]
please help me
I1030 19:42:55.136488 14723 solver.cpp:259]     Train net output #0: mbox_loss = 3.60342 (* 1 = 3.60342 loss)
I1030 19:42:55.136494 14723 sgd_solver.cpp:138] Iteration 19770, lr = 0.001
I1030 19:43:31.607108 14723 solver.cpp:243] Iteration 19780, loss = 2.88569
I1030 19:43:31.607174 14723 solver.cpp:259]     Train net output #0: mbox_loss = 2.70507 (* 1 = 2.70507 loss)
I1030 19:43:31.607180 14723 sgd_solver.cpp:138] Iteration 19780, lr = 0.001
I1030 19:44:03.203322 14723 solver.cpp:243] Iteration 19790, loss = 2.81859
I1030 19:44:03.203431 14723 solver.cpp:259]     Train net output #0: mbox_loss = 2.27487 (* 1 = 2.27487 loss)
I1030 19:44:03.203438 14723 sgd_solver.cpp:138] Iteration 19790, lr = 0.001
I1030 19:44:38.776464 14723 solver.cpp:243] Iteration 19800, loss = 2.81871
I1030 19:44:38.776559 14723 solver.cpp:259]     Train net output #0: mbox_loss = 2.05846 (* 1 = 2.05846 loss)
I1030 19:44:38.776566 14723 sgd_solver.cpp:138] Iteration 19800, lr = 0.001
I1030 19:45:13.684064 14723 solver.cpp:243] Iteration 19810, loss = 3.18381
I1030 19:45:13.684135 14723 solver.cpp:259]     Train net output #0: mbox_loss = 4.59389 (* 1 = 4.59389 loss)
I1030 19:45:13.684141 14723 sgd_solver.cpp:138] Iteration 19810, lr = 0.001
I1030 19:45:50.371204 14723 solver.cpp:243] Iteration 19820, loss = 3.41101
I1030 19:45:50.371302 14723 solver.cpp:259]     Train net output #0: mbox_loss = 2.43652 (* 1 = 2.43652 loss)
why i running the loss unstabitily and chanrge slow.
像这样是不是就无法收敛了呀。这到底是为什么呢。

**Comment by zoumengchao** [comment]
你这才1.9W 在大学习率下震荡是很正常不过了.  这个原始设置为分段学习率,高过8W以后就会调低一次,10W后再调低一次.   当然.你也有可能正好在半山腰中的一块平原上,能不能出来也不好说.
